src/main/java/sirius/biz/codelists/CodeListLookupTable.java

```diff
@@ -131,4 +131,9 @@ private boolean filter(CodeListEntry<?, ?> entry, String searchTerm, String lang
     protected Stream<LookupTableEntry> performScan(String lang) {
         return codeLists.getEntries(codeList).stream().map(entry -> extractEntryData(entry, lang));
     }
+
+    @Override
+    protected Stream<LookupTableEntry> performQuery(String lang, String lookupPath, String lookupValue) {
+        throw new UnsupportedOperationException();
```

Comment:
Maybe a a small comment why this is unsupported

mainly to differiantiate between "not yet implemented" and "this method will never work on this class"